code/datums/gamemode/factions/faction.dm

```diff
@@ -66,12 +66,12 @@ var/list/factions_with_hud_icons = list()
 /datum/faction/proc/HandleNewMind(var/datum/mind/M) //Used on faction creation
 	for(var/datum/role/R in members)
 		if(R.antag == M)
-			WARNING("Mind was already a role in this faction")
+			warning("Mind was already a role in this faction")
```

Comment:
Why would you change these from `WARNING` (which shows file, line number, src, usr) to `warning` (which only shows the message)?

Reply:
Good point